source/extensions/common/wasm/base_config.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <memory>
+
+#include "envoy/extensions/wasm/v3/wasm.pb.validate.h"
+
+#include "common/common/logger.h"
+
+#include "absl/strings/str_cat.h"
+#include "include/proxy-wasm/wasm.h"
+#include "include/proxy-wasm/wasm_vm.h"
+#include "include/proxy-wasm/word.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Common {
+namespace Wasm {
+
+class WasmBaseConfig {
```

Comment:
We always use `Base` to denote Proxy-Wasm class, i.e. `PluginBase` is defined in Proxy-Wasm, and `Plugin` is its subclass defined in Envoy, so this should be called `WasmConfig`, or perhaps `PluginConfig`?